spec/rubocop/cop/rails/select_map_spec.rb

```diff
@@ -45,7 +45,7 @@
     RUBY
   end
 
-  it 'registers an offense when using `select(:column_name).map(&:column_name)` without receiver model' do
+  it 'registers an offense when using `select(:column_name).map(&:column_name)` without receiver' do
```

Comment:
When I added "model" to the test name in L59, `Layout/LineLength` pointed it out, so I removed it from this test name as well to keep the format consistent.

```
spec/rubocop/cop/rails/select_map_spec.rb:59:121: C: Layout/LineLength: Line is too long. [124/120]
  it 'registers an offense when using `select(:column_name).where(conditions).map(&:column_name)` without receiver model' do
                                                                                                                        ^^^^
```

Reply:
I think keeping `git blame` intact is more valuable, but it’s unlikely that anything significant will happen here.